approval_polls/views.py

```diff
@@ -79,7 +79,7 @@ def add(request):
     if not len(choices):
       return render(request, 'approval_polls/error.html', {'error_message': 'At least one choice is required'})
 
-    p = Poll(question=question, pub_date=timezone.now())
+    p = Poll(question=question, pub_date=timezone.now(), username=request.POST['username'].strip())
```

Comment:
@ashimaathri 
Based on my understanding, the primary key of user is user_id, but what we want is username, so I think we should use to_field to retrieve the information.
Let me know if you have different thought.
Thanks!